protocol/contracts/beanstalk/storage/System.sol

```diff
@@ -389,9 +389,10 @@ struct Migration {
  * gaugePoint and liquidityWeight implementations.
  */
 struct Implementation {
-    address target;
+    address target; // 20 bytes
     bytes4 selector;
     bytes1 encodeType;
+    uint32 timeout; // 4 bytes
```

Comment:
we should make this a generic bytes so that future external implementations can have generic byte data to be added to its oracle (i.e a generalized oracle that can be used for multiple tokens.